### DISCUSSION TITLE
Pay out with Open Collective

### DISCUSSION THREAD

**Discussion by RCheesley** [stackaid/stackaid#7] [Q&A] [answered]
I'd prefer to both send and receive payments on Open Collective rather than pay more fees with Stripe. Would be helpful to have that as a feature.

**Comment by dudleycarr** [comment, marked as answer]
@RCheesley We currently pay out to projects on Open Collective for projects funded on StackAid but haven't registered with StackAid. The recipient project currently nets less than if they registered with StackAid since OC has its own fees.

If you are receiving money from StackAid via Stripe Connect, there are no added fees for us to transfer the money that has been allocated to you by StackAid subscribers.

**Comment by voxpelli** [comment]
Not sure how Stripe would result in higher fees. Open Collective hosts also have fees, the only time there are no fees is typically only when moving money from within projects of the same host there: https://twitter.com/voxpelli/status/1537166120804835330?s=20

And with eg the Open Source Collective projects actually promise to use the money to further their own project, so most of the time they can't really forward that money easily: https://twitter.com/voxpelli/status/1537169272132149248?s=20
